digits_test.go

```diff
@@ -266,6 +266,22 @@ var testCases = []*testCase{
 			tail:       "00,000.00)",
 		},
 	},
+	{
```

Comment:
this is a good test case 👍 

Now please include this test case: https://github.com/joshuanario/digits/issues/11#issuecomment-1817630173

Reply:
added, see issue comment